### PR TITLE
fixed password reset

### PR DESCRIPTION
**Pull request by edwardmccaughan**

### CODE REVIEW

File: app/views/devise/passwords/edit.haml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+    .text-center
+      %h2{:style => "margin-bottom:0px"} Passwort Andern
+  .row
+    .col-md-8.col-md-offset-2.col-xs-12.col-lg-6.col-lg-offset-3.col
+      = form_for(resource, as: resource_name, url: password_path(resource_name), html: { class: "form-horizontal margin-top-30", method: :put }) do |f|
+        = devise_error_messages!
+        = f.hidden_field :reset_password_token
+        
+        .form-group
+          = f.label :password, "Neue passwort", :class => "control-label col-sm-2"
```
Comment (jottr) [inline app/views/devise/passwords/edit.haml]: Please fix spelling. It should be "Neues Passwort."

File: app/views/devise/registrations/new.html.erb [hunk 2]
```diff
@@ -34,7 +34,7 @@
         </div>
 
         <div class="form-group">
-          <%= f.label :password_confirmation, :class => "col-sm-2 control-label" %>
+          <%= f.label :password_confirmation, "Passwort bestätigung", :class => "col-sm-2 control-label" %>
```
Comment (jottr) [inline app/views/devise/registrations/new.html.erb]: This should be "Passwortbestätigung"

Comment (edwardmccaughan) [inline app/views/devise/registrations/new.html.erb]: When it's all one word, it's too big to fit in the label space, do you want me to change the label column to col-sm-3 (and make the input fields smaller) ?

Comment (jottr) [inline app/views/devise/registrations/new.html.erb]: Yes please. Could you provide a screenshot of the design change? Thx

Comment (xhr15) [inline app/views/devise/registrations/new.html.erb]: Do whatever is needed to make "Passwortbestätigung" fit :)

**Comment by edwardmccaughan** [comment]
ok, I fixed those comments and I changed the register page slightly to have more space for the input labels:
image

**Comment by jottr** [comment]
Is there a way you can fix the alignment so it has the same spacing for all forms?
Also, please use the word "Passwort" not "Password". 
Thx.

**Comment by edwardmccaughan** [comment]
@jottr like this?:
image

**Comment by jottr** [comment]
Yes, perfect.

**Pull request merged**